code/src/App.js

```diff
@@ -1,9 +1,34 @@
-import React from 'react'
+import React from "react";
+import { Provider } from "react-redux";
+import { combineReducers, createStore, compose } from "@reduxjs/toolkit";
+import persistState from 'redux-localstorage';
```

Comment:
Local storage - did not read up on this before - but I did now. Of course this is a very crucial functionality to make the todo list usable. I wish I was aware of that before. Great investigation to add this